client/settings/disable-upe-modal/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { useContext, useEffect } from 'react';
+
+import { __ } from '@wordpress/i18n';
+import { dispatch } from '@wordpress/data';
+import { Button, Dashicon, ExternalLink } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import ConfirmationModal from 'components/confirmation-modal';
+import PaymentMethod from 'components/payment-methods-list/payment-method';
+import useIsUpeEnabled from 'settings/wcpay-upe-toggle/hook';
+import WcPayUpeContext from 'settings/wcpay-upe-toggle/context';
+
+const NeedHelpBarSection = () => {
+	return (
+		<div className="disable-modal-help-notice">
+			<Dashicon className="disable-help-icon" icon="info-outline" />
+			<p>
+				{ interpolateComponents( {
+					mixedString: __(
+						'Need help? Visit {{ docsLink /}} or {{supportLink /}}.',
+						'woocommerce-payments'
+					),
+					components: {
+						docsLink: (
+							// eslint-disable-next-line max-len
+							<ExternalLink href="https://docs.woocommerce.com/document/payments/additional-payment-methods/#introduction">
+								{ __(
+									'WooCommerce Payments docs',
+									'woocommerce-payments'
+								) }
+							</ExternalLink>
+						),
+						supportLink: (
+							// eslint-disable-next-line max-len
+							<ExternalLink href="https://woocommerce.com/contact-us/">
+								{ __(
+									'contact support',
+									'woocommerce-payments'
+								) }
+							</ExternalLink>
+						),
+					},
+				} ) }
+			</p>
+		</div>
+	);
+};
+
+const DisableUPEModalBody = ( { enabledMethods } ) => {
```

Comment:
Still seeing `card` in this modal - this ensures that it is filtered out, without having to remove it from the main list:
```
const DisableUPEModalBody = ( { enabledMethods } ) => {
	const upeMethods = enabledMethods.filter( ( method ) => 'card' === method );

	return (
		<>
			<p>
				{ __(
					// eslint-disable-next-line max-len
					'Without the new payments experience, your customers will only be able to pay using credit card / debit card. You will not be able to add other sales-boosting payment methods anymore',
					'woocommerce-payments'
				) }
			</p>
			{ 0 < upeMethods.length ? (
				<>
					<p>
						{ __(
							'Payment methods that require the new payments experience:',
							'woocommerce-payments'
						) }
					</p>
					<ul>
						{ enabledMethods.map( ( { id, label, Icon } ) => (
							<PaymentMethod
								key={ id }
								Icon={ Icon }
								label={ label }
							/>
						) ) }
					</ul>
				</>
			) : null }
			<NeedHelpBarSection />
		</>
	);
};
```